docs/en/Running-Deep-Learning-on-Alluxio.md

```diff
@@ -0,0 +1,91 @@
+
+![Fuse]({{site.data.img.fuse}})
+
+Finally, Alluxio also provides local caching of frequently used data. This is particularly useful when the data is remote from the computation, such as the disaggregate compute from storage environment. Since Alluxio can cache the data locally, network IO is not incurred for accessing the data, so deep learning training can be more cost effective, and take less time.
+
+## Setting up Alluxio FUSE
+
+In this section, we will follow the instructions in the [FUSE section](Mounting-Alluxio-FS-with-FUSE.html) to setup FUSE, access training data of ImageNet in S3, and allow deep learning frameworks to access the data through FUSE.
+
+First, we’ll create a folder at the root in Alluxio
+
+```bash
+$ ./bin/alluxio fs mkdir /training-data
+```
+
+Then, we can mount the ImageNet data stored in an S3 bucket into path /training-data/imagenet, assuming the data is at the s3 path `s3a://alluxio-tensorflow-imagenet/`.
+
+```bash
+$ ./bin/alluxio fs mount /training-data/imagenet/ s3a://alluxio-tensorflow-imagenet/ --option aws.accessKeyID=<ACCESS_KEY_ID> --option aws.secretKey=<SECRET_KEY>
+```
+
+Note this command takes options to pass the S3 credentials of the bucket. And these credentials are associated with the mounting point, so that the future accesses won’t need the credentials.
+
+Next, we’ll start the Alluxio-FUSE process. First, we’ll create a folder `/mnt/fuse`, change its owner to the current user (ec2-user in this doc), and make it read-write. 
+
+```bash
+$ sudo mkdir -p /mnt/fuse
+$ sudo chown ec2-user:ec2-user /mnt/fuse
+$ chmod 664 /mnt/fuse
+```
+
+Next, we’ll run the Alluxio-FUSE shell to mount Alluxio folder training-data to the local folder /mnt/fuse.
+
+```bash
+$ ./integration/fuse/bin/alluxio-fuse mount /mnt/fuse /training-data
+```
+
+Now the FUSE process should have started, and you can check its status with:
+
+```bash
+$ ./integration/fuse/bin/alluxio-fuse stat
+```
+
+Now you can go the mounted folder and browse the data, it should display the training data stored in the cloud.
+
+```bash
+$ cd /mnt/fuse
+$ ll
+```
+
+This folder is ready for the deep learning frameworks to use, which would treat the Alluxio storage as if it’s a local folder. We’ll use this folder for the Tensorflow training in the next section.
+
+## Using Tensorflow on Alluxio FUSE
+
+We use Tensorflow as an example deep learning framework in this doc to show how Alluxio can help data access and management. To access the training data in S3 via Alluxio, with the Alluxio FUSE, we can simply pass the path /mnt/fuse/imagenet to the parameter data_dir of the benchmark script tf_cnn_benchmkars.py.
```

Comment:
This portion is a bit abstract, perhaps link or include the tensorflow benchmark script?